rare/models/launcher.py

```diff
@@ -15,7 +16,7 @@ class BaseModel:
     app_name: str
 
     @classmethod
-    def from_json(cls, data: dict):
+    def from_json(cls, data: Dict):
```

Comment:
Because that is how we use that in the rest of Rare. It is a remnant from the time we also supported python 3.8. IIRC it didn't have types in the same way later versions do.